### DISCUSSION TITLE
Pausing/Unpausing Should include Blocks

### DISCUSSION THREAD

**Discussion by ihategravel2** [Mojang/minecraft-editor#17] [Feedback]
Right now there  are "Pause Actors" and "Unpause Actors" buttons to.. pause the world. What I'm suggesting is this be expanded to blocks as well as entities. This would make it easier for builds that use blocks such as lava, water, flowers, leaves, etc without them breaking or flowing right away. When you unpause or export the world the blocks would do their thing as normal.

**Comment by JannisX11** [comment]
I agree with this. I know this was not initially implemented because of concerns that worlds would be saved with unstable blocks. To solve this, it could be required to unpause blocks before saving the world. Or there could be a warning dialog.

I think that is a much better and easier solution than trying to track the indirect effects of edits (such as falling gravel) as part of the undo save. Keeping track of side effects only solves some of the issues, and it's difficult because it might affect blocks across many chunks.

**Reply by minecraftchloe** [reply]
❤️ Great response and also a great idea, Jannis! Thank you!!

**Comment by minecraftchloe** [comment]
Definitely on our radar to expand pause to include blocks!
